sources/manager_file.py

```diff
@@ -28,7 +28,8 @@ def load_localization(file: str):
 
         :param file: Localization file path, related to current file (in sources root).
         """
-        with open(join("sources", file), encoding="utf-8") as config_file:
+        translation_path = join("sources", file) if EM.DEBUG_RUN else file
```

Comment:
with open(join(dirname(__file__), file) 
How about adding like this instead of if condition